policyengine_us/tests/policy/baseline/gov/states/or/oha/healthier_oregon/integration.yaml

```
@@ -0,0 +1,66 @@
+# Integration tests for Oregon Healthier Oregon with cost calculation
+
+- name: Undocumented adult gets healthcare benefit value
+  period: 2024
+  absolute_error_margin: 1
```

```suggestion
  absolute_error_margin: 0.1
```
The test should still pass